pom.xml

```diff
@@ -30,7 +30,7 @@
 		<logback.version>1.1.11</logback.version>
 		<selenium.version>3.0.0</selenium.version>
 		<build.server.version>1.2.0</build.server.version>
-		<git.server.version>1.0.10</git.server.version>
+		<git.server.version>1.0.11-SNAPSHOT</git.server.version>

```

Comment:
Yup I am aware of it. But this branch needs git-server changes. I'll review those first and we'll release a new version. CI should pass afterwards.